src/PhpSpec/Listener/MethodNotFoundListener.php

```diff
@@ -70,7 +70,8 @@ public function afterExample(ExampleEvent $event)
             return;
         }
 
-        $classname = get_class($exception->getSubject());
+        $subject = $exception->getSubject();
+        $classname = null === $subject ? get_class() : get_class($subject);
```

Comment:
No use case. It should never happen as the method is guarded to only exceptions of method not found, which means subject is an object.